src/main/java/com/checkmarx/flow/controller/IastController.java

```diff
@@ -39,20 +55,40 @@
                 .build());
     }
 
-    @PostMapping(value = {"/stop-scan-and-create-jira-issue/{scanTag}"})
-    public ResponseEntity<EventResponse> stopScanAndCreateJiraIssue(
+    @PostMapping(value = {"/stop-scan-and-create-{tracker}-issue/{scanTag}"})
+    public ResponseEntity<EventResponse> stopScanAndCreateIssue(
             @PathVariable(value = "scanTag", required = false) String scanTag,
-            @RequestHeader(value = TOKEN_HEADER) String token) {
+            @PathVariable(value = "tracker", required = false) String bugTrackerName,
+            @RequestHeader(value = TOKEN_HEADER) String token,
```

Comment:
@AvivCx one question about this header name (X-Atlassian-Token), when I'm performing a request to another bug tracker, for instance azure, it makes sense to have this header "X-Atlassian-Token" regarding that is related to Jira?

I saw that token is the CxFlow API token, in this case the header name should be more generic, right? 
(some changes that I have here on this PR was merged from GitHub branch - pr-iast-2245-add_iast_integration_with_github)